src/metadata/bpm.cpp

```diff
@@ -0,0 +1,60 @@
+#include "metadata/bpm.h"
+
+namespace Mixxx {
```

Comment:
I see that you've started putting a namespace around everything. I'd like to revisit whether we need this considering the plan to remove the Source plugin system.